### ISSUE TITLE
passing a custom loss function to model.compile()

### ISSUE THREAD

**Issue by RoniGurevich** [fchollet/keras#4926]
in keras/objectives.py
I replaced
```python
def get(identifier):
    return get_from_module(identifier, globals(), 'objective')
```
with:
```python
def get(identifier):
    if callable(identifier):
        return identifier
    return get_from_module(identifier, globals(), 'objective')
```
now when I call model.compile() I can pass it my own loss function and use it while training instead of the predefined ones in objectives.py

thought I would share to help others

**Comment by valtron** [comment]
That shouldn't be necessary: [`get_from_module`](/fchollet/keras/blob/master/keras/utils/generic_utils.py#L10) already does that.
